### PR TITLE
media-downloader: Add version 4.7.0

### PR DESCRIPTION
**Pull request by ahmed-0011**
Add [media-downloader](https://github.com/mhogomchungu/media-downloader), a Qt/C++ front end to yt-dlp, youtube-dl, gallery-dl, lux, you-get, svtplay-dl, aria2c, wget and safari books..

Closes https://github.com/ScoopInstaller/Extras/issues/11167

* [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aliesbelik** [comment]
Relates to mhogomchungu/media-downloader#266.

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Formatting & extract_dir autoupdate.

File: bucket/media-downloader.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "4.7.0",
+    "description": "A Qt/C++ front end to yt-dlp, youtube-dl, gallery-dl, lux, you-get, svtplay-dl, aria2c, wget and safari books..",
+    "homepage": "https://github.com/mhogomchungu/media-downloader",
+    "license": "GPL-2.0",
+    "url": "https://github.com/mhogomchungu/media-downloader/releases/download/4.7.0/MediaDownloader-4.7.0.zip",
+    "hash": "181527e400b672a36e5b652ae9fd03f5df7c9a8d5b0933c48b003b7856410387",
+    "extract_dir": "MediaDownloader-4.7.0",
+    "bin": "media-downloader.exe",
+    "shortcuts": [["media-downloader.exe", "media-downloader"]],
```
Comment (aliesbelik) [inline bucket/media-downloader.json:10]:
```suggestion
    "shortcuts": [
        [
            "media-downloader.exe",
            "Media Downloader"
        ]
    ],
```

File: bucket/media-downloader.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/mhogomchungu/media-downloader/releases/download/$version/MediaDownloader-$version.zip"
```
Comment (aliesbelik) [inline bucket/media-downloader.json:13]:
```suggestion
        "url": "https://github.com/mhogomchungu/media-downloader/releases/download/$version/MediaDownloader-$version.zip",
        "extract_dir": "MediaDownloader-$version"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Formatting & extract_dir autoupdate.

**Comment by ahmed-0011** [comment]
\verify

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**